runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -530,6 +530,7 @@ public void mouseClicked(MouseEvent e)
 						final JFrame parent = new JFrame();
 						JColorChooser jColorChooser = new JColorChooser(existingColor);
 						jColorChooser.getSelectionModel().addChangeListener(e1 -> colorPicker.setBackground(jColorChooser.getColor()));
+						jColorChooser.getSelectionModel().addChangeListener(e1 -> colorPicker.setText("#" + Integer.toHexString(jColorChooser.getColor().getRGB()).substring(2).toUpperCase()));
```

Comment:
Just set both background and text in same listener

Reply:
I couldn't figure out how to do that with the lambda, sorry.